WalletWasabi.Gui/Program.cs

```diff
@@ -5,11 +5,13 @@
 using NBitcoin;
 using System;
 using System.IO;
+using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 using System.Threading.Tasks;
 using WalletWasabi.Gui.CommandLine;
 using WalletWasabi.Gui.ViewModels;
 using WalletWasabi.Logging;
+[assembly:InternalsVisibleTo("WalletWasabi.Tests")]
```

Comment:
Please remove this and make the internal public, we don't use internals, only by accident.

Reply:
I think this is wrong. Making tests internal is a practice used widely by Microsoft themselves. (https://github.com/aspnet/AspNetCore/search?q=InternalsVisibleTo&unscoped_q=InternalsVisibleTo)